main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <iomanip>
+
+int main() {
+
+    const int SHARES_BOUGHT = 1000;
+    const double PURCHASE_PRICE = 45.50;
+    const double COMMISSION_RATE = .02;
+    const int SHARES_SOLD = 1000;
+    const double SALE_PRICE = 56.90;
+    double amount_paid = SHARES_BOUGHT * PURCHASE_PRICE;
+    double first_commission = amount_paid * COMMISSION_RATE;
+    double amount_sold = SHARES_SOLD * SALE_PRICE;
+    double second_commission = amount_sold * COMMISSION_RATE;
+    double profit = amount_sold - amount_paid - first_commission - second_commission;
+
+    std::cout << std::fixed << std::setprecision(2) << "Joe bought " << SHARES_BOUGHT << " shares at $"
```

Comment:
If this `<< std::fixed << std::setprecision(2)` were on their own line it would make the intent more clear and keep this line as short as it has to be while still doing its job.